### PR TITLE
Fix(pn 13314)/Fix(PN-13353)/FIX(PN-13324)

### PR DESCRIPTION
**Pull request by leleOFA**
## Short description
fix regex for subject when select additional lang
fix bug on compile newNotificationState when the Italian language is chosen again

## List of changes proposed in this pull request
- insert new control of yup form for subject
- fix copy  of length of subject
- insert reset additionalLang/additionalSubject when select ita lang

## How to test
 pn 13314 : In pa  when I create the notification and enter a second language I can insert a string of up to 67 characters in the notification subject

 PN-13353 : when creating notifications, after inserting a second language and an additional object, returning to the single language in Italian resets the second object and the previously selected language
 
 pn 13324 : change separator of subj and add subj

### CODE REVIEW

File: packages/pn-pa-webapp/src/components/NewNotification/PreliminaryInformationsContent.tsx [hunk 1]
```diff
@@ -9,13 +9,17 @@ import { NewNotificationLangOther } from '../../models/NewNotification';
...
+};
+
+function SubjectFocusHelperText({hasOtherLang} : SubjectFocusHelperTextProps) {
   const { t } = useTranslation(['common']);
   const { focused } = useFormControl() || {};
 
   return useMemo(() => {
     if (focused) {
-      return t('too-long-field-error', { maxLength: 134 });
+      return t('too-long-field-error', { maxLength: hasOtherLang ? 67: 134 });
```
[the remark this answers is not in the document]
Comment (leleOFA) [reply packages/pn-pa-webapp/src/components/NewNotification/PreliminaryInformationsContent.tsx]: done

**Review by AndreaCimini90 — APPROVED** [review]

File: packages/pn-pa-webapp/src/components/NewNotification/PreliminaryInformations.tsx [hunk 2]
```diff
@@ -148,11 +153,17 @@ const PreliminaryInformations = ({ notification, onConfirm }: Props) => {
...
   const handleChangeTouched = async (e: ChangeEvent) => {
+    
     formik.handleChange(e);
     await formik.setFieldTouched(e.target.id, true, false);
   };
 
   const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const value = e.target.value;
+    if(value === 'it'){
+      void formik.setFieldValue('additionalLang','');
```
Comment (AndreaCimini90) [inline packages/pn-pa-webapp/src/components/NewNotification/PreliminaryInformations.tsx:164]: This code partially solves the bug. It empties the fields but it doesn't reset the touched state. Use the formik.setValues to set additionalLang and additionalSubject in one function and set the second parameter to false to skip the validation. Use formik.setFieldTouched to set the touched state of additionalSubject.

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

File: packages/pn-pa-webapp/src/components/NewNotification/PreliminaryInformations.tsx [hunk 3]
```diff
@@ -148,11 +153,17 @@ const PreliminaryInformations = ({ notification, onConfirm }: Props) => {
   });
 
   const handleChangeTouched = async (e: ChangeEvent) => {
+
```
[the remark this answers is not in the document]
Comment (leleOFA) [reply packages/pn-pa-webapp/src/components/NewNotification/PreliminaryInformations.tsx:156]: done

File: packages/pn-pa-webapp/src/components/NewNotification/PreliminaryInformations.tsx (hunk 2 above)
Comment (leleOFA) [reply packages/pn-pa-webapp/src/components/NewNotification/PreliminaryInformations.tsx]: doijne

**Review by AndreaCimini90 — APPROVED** [review]

**Pull request merged**